x-pack/plugins/siem/server/graphql/kpi_hosts/resolvers.test.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { GraphQLResolveInfo } from 'graphql';
+
+import { Source } from '../../graphql/types';
+import { FrameworkRequest, internalFrameworkRequest } from '../../lib/framework';
+import { KpiHosts } from '../../lib/kpi_hosts';
+import { KpiHostsAdapter } from '../../lib/kpi_hosts/types';
+import { SourceStatus } from '../../lib/source_status';
+import { Sources } from '../../lib/sources';
+import { createSourcesResolvers } from '../sources';
+import { SourcesResolversDeps } from '../sources/resolvers';
+import { mockSourcesAdapter, mockSourceStatusAdapter } from '../sources/resolvers.test';
+
+import { mockKpiHostsData } from './kpi_hosts.mock';
+import { createKpiHostsResolvers, KpiHostsResolversDeps } from './resolvers';
+
+const mockGetKpiHosts = jest.fn();
+mockGetKpiHosts.mockResolvedValue({
+  KpiHosts: {
+    ...mockKpiHostsData.KpiHosts,
+  },
+});
+const mockKpiHostsAdapter: KpiHostsAdapter = {
+  getKpiHosts: mockGetKpiHosts,
+};
+
+const mockKpiHostsLibs: KpiHostsResolversDeps = {
+  kpiHosts: new KpiHosts(mockKpiHostsAdapter),
+};
+
+const mockSrcLibs: SourcesResolversDeps = {
+  sources: new Sources(mockSourcesAdapter),
+  sourceStatus: new SourceStatus(mockSourceStatusAdapter, new Sources(mockSourcesAdapter)),
+};
+
+const req: FrameworkRequest = {
+  [internalFrameworkRequest]: {
+    params: {},
+    query: {},
+    payload: {
+      operationName: 'test',
+    },
+  },
+  params: {},
+  query: {},
+  payload: {
+    operationName: 'test',
+  },
+};
+
+const context = { req };
+
+describe('Test Source Resolvers', () => {
```

Comment:
Since you added API Integration tests (awesome!) these `resolvers.test.ts` tests are no longer necessary -- feel free to remove this entire file :)